core/trino-main/src/test/java/io/trino/type/AbstractTestType.java

```diff
@@ -203,41 +202,6 @@ public void testBlock()
         }
     }
 
-    @Test
-    public void testRange()
-    {
-        assertThat(type.getRange())
-                .isEmpty();
-    }
-
-    @Test
-    public void testPreviousValue()
-    {
-        Object sampleValue = getSampleValue();
-        if (!type.isOrderable()) {
-            assertThatThrownBy(() -> type.getPreviousValue(sampleValue))
```

Comment:
Would it make sense to at least cc the author of this code on the pr that fixes it?

Reply:
It was mentioned in this thread, which you were already part of: https://github.com/trinodb/trino/pull/18588#discussion_r1288304618